<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>MSC|Goddard Group</title>
    <link rel="stylesheet" href="css/basic.css">
    <link rel="stylesheet" href="css/Home.css">
    <link rel="stylesheet" href="css/slide.css">
    <link rel="stylesheet" href="css/about.css">
    <link rel="stylesheet" href="css/member.css">    
    <link rel="stylesheet" href="css/research.css">
    <link rel="stylesheet" href="css/publication.css">
    <link rel="stylesheet" href="css/table.css">
    <link rel="stylesheet" href="css/photo.css">
    <link rel="stylesheet" href="css/contact.css">    
    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700;900&display=swap" rel="stylesheet">
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.3/jquery.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/Sly/1.6.1/sly.min.js"></script>
    <load target="assets/js/sly.min.js" type="header" />
    <load target="//cdnjs.cloudflare.com/ajax/libs/jquery/1.12.3/jquery.min.js" />
    <load target="//cdnjs.cloudflare.com/ajax/libs/Sly/1.6.1/sly.min.js" />
</head>

<body>

<!-- Logo& Navigation -->
<header class="top">
    <div class="top-container container">
        <div class="logo">
            <a href="Home.html">
            <img 
                alt="MSC Materials and Process Simulation Center and Goddard Group of Caltech" 
                src="img/logo-03.svg"></a>            
        </div>
        
        <!-- Menu Icon for Mobile -->
        <div class="mobile">
            <img class="icon" src="img/menu-open.svg" >
               
        </div>


        <!-- Navigation menu -->
        
        <div class="nav" id="nav">
        <!-- <div id="modal"> -->
            <div class="wiki">
                <a class="wiki-btn" href="https://wiki.wag.caltech.edu/twiki/bin/view" target="_blank">WIKI</a>
            </div>
            <div class="nav-menu">
                <a href="AboutWAG.html">About</a>
                <a href="MembersCurrent.html">Members</a>
                <a id="nav-menu-on" href="ResearchOverview.html">Research</a>
                <a href="Publications.html">Publications</a>
                <a href="EventsPhotos.html">Events</a>
                <a href="Contact.html">Contact</a>
            </div>
        <!-- </div> -->
        </div>
        
    </div>
</header>

<!-- Sub menues -->

<div class="sub-menus">
    <div class="sub-container container" >
        <div class="sub-innerbox innerbox" id="wrapper">
            <div class="pre-arrow"></div>
            <div class="slide-frame">
                <ul class="slidee">
                    <li><a href="ResearchOverview.html">Overview</a></li>
                    <li><a href="ResearchA.html">Methods</a></li>
                    <li><a href="ResearchB.html">Materials Applications</a></li>
                    <li><a id="sub-menu" href="ResearchC.html">Biotechnology and Drug Design</a> </li>
                    <li><a href="ResearchD.html">Mechanical Bonded Devices</a></li>
                </ul>             
           
            </div>
            <div class="next-arrow"></div>
        </div>
    </div>
</div>

<!-- Main Contents -->
<main class="main-content">
    <div class="main-container container">
        <div class="main-innerbox innerbox">


            <!-- From Here!! -->

            <div class="main">

                <div class="focus-menu">
                    <div class="focus-wrap">
                        <div class="focus-label">C1</div>
                        <div class="focus-title">G-Protein Coupled Receptors (GPCRs)</div>
                        <div class="focus-btn">Learn More</div>
                    </div>
                    <div class="focus-text">
                        <span class="list-sq"></span><span class="sub-title">C1-1. Class A</span>
                        <ul>
                            <li>C1-1-1. Opioid receptors: μ-OR, κ−OR, coupling to G-Protein
                                <p class="papers">#1480. 
                                    <a href="https://www.cambridge.org/core/journals/qrb-discovery/article/g-proteinfirst-activation-mechanism-of-opioid-receptors-by-gi-protein-and-agonists/23FF9692A7DBEDB039A98F9CB612B52D" target="_blank">
                                    <i> "The G protein-first activation mechanism of opioid receptors by Gi protein and agonists"</i></a>. 
                                </p>
                            </li>
                            <li>C1-1-2. Activation mechanism class A
                                <p class="papers">#1518. 
                                    <a href="https://www.pnas.org/doi/full/10.1073/pnas.2110085119" target="_blank">
                                    <i> "The mechanism for ligand activation of the GPCR–G protein complex"</i></a>. 
                                </p>
                                <p>We established  the G-Protein (GP) first mechanism for G-Protein Coupled Receptors (GPCRs) in which the unactivated G-Protein with GDP attached binds first inactive GPCR to form a precoupled GPCR-GP complex with intracellular region partially coupled but not activated. Then binding of agonist causes the Ga part of the GP to open releasing GDP for GP signaling. It is this precoupled structure that is relevant to drug design, but there are not yet any crystal structures (all current structures are for the fully activated agonist-GPCR-GP structure after activation and release of the GDP. </p>
                                <img alt="figure 1 of paper number 1518"
                                    src="img/research/1518a.png"
                                    width="70%">
                                <p>Binding of a ligand to this precoupled structure can lead to full activation (full agonist), to partial activation (partial agonist), or to no activation (inverse agonist) depending on where the ligand binds to the precoupled GPCR-GP complex. </p>
                                <img alt="figure 2 of paper number 1518"
                                    src="img/research/1518b.png"
                                    width="70%">
                                    <p class="figure">Figure 1518. Precoupled GPCR-GP state plays key role in activation. Binding agonist opens up Gα</p>
                                <img alt="figure 3 of paper number 1518"
                                    src="img/research/1518c.png"
                                    width="70%">
                                    <p class="figure">Figure 1518. New mechanisim for GPCR activation: GP binds, breaks ionic lock, bonds to 3 ICLs, ortents Gα5 helix. Then agonist binds, activates GP with Gα5 moving further into the GPCR</p>
                                
                            </li><br>
                            <li>C1-1-3. β-arrestin activation
                                <p class="papers">#1382. 
                                    <a href="https://www.pnas.org/doi/full/10.1073/pnas.1918264117" target="_blank">
                                    <i> "Mechanism of β-arrestin recruitment by the μ-opioid G protein-coupled receptor"</i></a>. 
                                </p>
                            </li>
                            <li>C1-1-4. Serotonin 1E β-arrestin activation for neuroprotection
                                <p class="papers">#1503. 
                                    <a href="https://link.springer.com/article/10.1007/s00018-021-04021-3" target="_blank">
                                    <i> "Novel interaction between neurotrophic factor-α1/carboxypeptidase E and serotonin receptor, 5-HTR1E, protects human neurons against oxidative/neuroexcitotoxic stress via β-arrestin/ERK signaling"</i></a>. 
                                </p>
                            </li>
                            <li>C1-1-5. Agonist-κ-OR-GP
                                <p class="papers">#1371. 
                                    <a href="https://www.pnas.org/doi/full/10.1073/pnas.1910006117" target="_blank">
                                    <i> "The atomistic level structure for the activated human κ-opioid receptor bound to the full Gi protein and the MP1104 agonist"</i></a>. 
                                </p>
                            </li>
                            <li>C1-1-6. Adrenergic receptors: β2-AR
                                <p class="papers">#1510. 
                                    <a href="https://www.mdpi.com/2075-4426/12/3/331" target="_blank">
                                    <i> "Biased β-Agonists Favoring Gs over β-Arrestin for Individualized Treatment of Obstructive Lung Disease"</i></a>. 
                                </p>
                                <p class="papers">#1478. 
                                    <a href="https://www.pnas.org/doi/full/10.1073/pnas.2026668118" target="_blank">
                                    <i> "Identification and characterization of an atypical Gαs-biased β2AR agonist that fails to evoke airway smooth muscle cell tachyphylaxis"</i></a>. 
                                </p>
                            </li>
                            <li>C1-1-7. Olfactory receptors
                                <p class="papers">#1435. 
                                    <a href="https://link.springer.com/chapter/10.1007/978-3-030-18778-1_37" target="_blank">
                                    <i> " Structural Variation and Odorant Binding for Olfactory Receptors Selected from the Six Major Subclasses of the OR Phylogenetic Tree"</i></a>. 
                                </p>
                            </li>
                        </ul>
                        
                        <span class="list-sq"></span><span class="sub-title">C1-2. Class T: Taste receptors (bitter)</span>
                        <ul>
                            <li>C1-2-1. TAS2R4
                                <p class="papers">#1465. 
                                    <a href="https://www.cambridge.org/core/journals/qrb-discovery/article/predicted-structure-of-fully-activated-human-bitter-taste-receptor-tas2r4-complexed-with-g-protein-and-agonists/252D0363D3638C89D654D6B5AB4A1CD5" target="_blank">
                                    <i> "Predicted structure of fully activated human bitter taste receptor TAS2R4 complexed with G protein and agonists"</i></a>. 
                                </p>
                            </li>
                            <li>C1-2-2. TAS2R5
                                <p class="papers">#1489. 
                                    <a href="https://pubs.acs.org/doi/10.1021/acs.jpclett.1c02162" target="_blank">
                                    <i> "Structures and Agonist Binding Sites of Bitter Taste Receptor TAS2R5 Complexed with Gi Protein and Validated against Experiment"</i></a>. 
                                </p>
                            </li>
                            <li>C1-2-3. TAS2R14
                            </li>
                        </ul>
                        
                        <span class="list-sq"></span><span class="sub-title">C1-3. Class C Dimer GPCRs</span>
                        <ul>
                            <li>C1-3-1. Heterodimer Sweet Receptor: TAS1R2/1R3
                                <p class="papers">#1252. 
                                    <a href="https://www.pnas.org/doi/full/10.1073/pnas.1700001114" target="_blank">
                                    <i> "Activation mechanism of the G protein-coupled sweet receptor heterodimer with sweeteners and allosteric agonists"</i></a>. 
                                </p>
                            </li>
                            <li>C1-3-2. Sweet Heterodimer Synergistic Effects
                                <p class="papers">#1492. 
                                    <a href="https://pubs.acs.org/doi/10.1021/acs.jafc.1c03779" target="_blank">
                                    <i> "Synergic Effects in the Activation of the Sweet Receptor GPCR Heterodimer for Various Sweeteners Predicted Using Molecular Metadynamics Simulations"</i></a>. 
                                </p>
                            </li>
                            <li>C1-3-3. Homodimer Sweet Activation
                                <p class="papers">#1491. 
                                    <a href="https://pubs.acs.org/doi/10.1021/jacs.1c08839" target="_blank">
                                    <i> "Predicted Structure of Fully Activated Tas1R3/1R3′ Homodimer Bound to G Protein and Natural Sugars: Structural Insights into G Protein Activation by a Class C Sweet Taste Homodimer with Natural Sugars"</i></a>. 
                                </p>
                            </li>
                            <li>C1-3-4. GABA receptor
                                <p class="papers">#1526. 
                                    <a href="https://www.nature.com/articles/s41467-022-32213-3" target="_blank">
                                    <i> "G protein coupling and activation of the metabotropic GABAB heterodimer"</i></a>. 
                                </p>
                            </li>
                        </ul>

                        <span class="list-sq"></span><span class="sub-title">C1-4. Class B
                        </span>
                        <ul>
                            <li>C1-4-1. GLP1-R
                                <p class="papers">#1000. 
                                    <a href="https://www.pnas.org/doi/full/10.1073/pnas.1218051109" target="_blank">
                                    <i> "Predicted structure of agonist-bound glucagon-like peptide 1 receptor, a class B G protein-coupled receptor"</i></a>. 
                                </p>
                            </li>
                    </div>
                </div>

                <div class="focus-menu">
                    <div class="focus-wrap">
                        <div class="focus-label">C2</div>
                        <div class="focus-title">Nucleic Acids</div>
                        <div class="focus-btn">Learn More</div>
                    </div>
                    <div class="focus-text">
                        <span class="list-sq"></span><span class="sub-title">C2-1. Conditional siRNA Therapeutics
                        </span>
                        <ul>
                            <li class="papers">#1508. 
                                    <a href="https://www.sciencedirect.com/science/article/pii/S216225312200004X?via%3Dihub" target="_blank">
                                    <i> "G protein coupling and activation of the metabotropic GABAB heterodimer"</i></a>. 
                            </li><br>
                            <img alt="figure of the paper no.1508"
                                    src="https://ars.els-cdn.com/content/image/1-s2.0-S216225312200004X-gr1_lrg.jpg"
                                    width="95%">
                            <p class="figure">Figure 1508. Design, operating concept, and chemical modifications of Cond-siRNAs
                                        (A) The secondary and tertiary structure of Cond-siRNAs. Docking of the tertiary structure model to the X-ray crystal structure of Giardia Dicer shows that the sensor can sterically block RNAi pathway enzymes from binding to the siRNA. This prevents RNAi processing in the OFF state. (B) RNAi activation via strand displacement. When a complementary input RNA meets the Cond-siRNA (I), the input forms a toe with a 30 or 50 single-stranded overhang on the sensor strand (II), leading to strand displacement (III). Displacement results in release of the siRNA (IV). Cellular nucleases remove the core-strand overhang siRNA (V), leaving an active siRNA that is free to enter the RNAi pathway (VI). (C) Chemical modifications that improve OFF/ON RNAi switching in mammalian cells include (1) PS modifications in the sensor overhang, (2) LNA and 20OMe modifications in the base-paired portion of the sensor strand (3) 20OMe modifications at the 50 and-30 termini of the core strand (highlighted in yellow), (4) PS backbone and base modifications at the ends of the
                                        siRNA to block exonucleases, (5) LNA modifications to improve Tm of the siRNA, (6) and base modifications in the siRNA duplex. 
                            </p>
                            <p>The fundamental paper on conditional-siRNA (COSI) technology. We report here that we have validating a simple and programmable new design that functions reliably in mammalian cells. We show that these conditionally activated siRNAs (Cond-siRNAs) can switch RNAi activity against different targets between clearly distinguished OFF and ON states in response to different cellular RNAbiomarkers. one version of our construct demonstrated biologically meaningful inhibition of a heart-disease-related target gene protein phosphatase 3 catalytic subunit alpha (PPP3CA) in response to increased expression of the pathological marker atrial natriuretic peptide (NPPA) messenger RNA (mRNA). Our results demonstrate the ability of synthetic riboswitches to regulate gene expression in mammalian cells, opening a new path for development of programmable siRNA pro-drugs. </p>
                        </ul><br>

                        <span class="list-sq"></span><span class="sub-title">C2-2. Review of MSC Studies on DNA-RNA Structures
                        </span>
                        <ul>
                            <li class="papers">#1464. 
                                    <a href="https://link.springer.com/chapter/10.1007/978-3-030-18778-1_69" target="_blank">
                                    <i> " DNA-RNA"</i></a>. 
                            </li>
                            <p>Summary of MSC research on DNA and RNA systems up to ~2019.</p>
                        </ul><br>

                        <span class="list-sq"></span><span class="sub-title">C2-3. DNA Crossover Structures; Collaboration with Ned Seeman, NYU
                        </span>
                        <ul>
                            <li class="papers">#1318. 
                                <a href="https://pubs.acs.org/doi/10.1021/acs.biochem.8b01148" target="_blank">
                                <i> " The PX Motif of DNA Binds Specifically to Escherichia coli DNA Polymerase I"</i></a>. 
                            </li>
                            <li class="papers">#1182. 
                                <a href="https://pubs.rsc.org/en/content/articlelanding/2016/NR/C6NR03418G" target="_blank">
                                <i> " Dramatic changes in DNA conductance with stretching: structural polymorphism at a critical extension"</i></a>. 
                            </li>
                            <li class="papers">#995. 
                                <a href="https://pubs.acs.org/doi/10.1021/jp306473u" target="_blank">
                                <i> " Role of Specific Cations and Water Entropy on the Stability of Branched DNA Motif Structures"</i></a>. 
                            </li><br>
                            <div class="gather-imgs">
                                <div class="gather-img">
                                    <img alt="figure of paper no.995"
                                        src="img/research/995.png"
                                        width="98%">
                                </div>
                                <div class="gather-img">
                                    <p class="figure">Figure 995. Free energy and entropy of stabilization of DNA 3-way junction.</p>
                                </div>
                            </div><br>
                            <li class="papers">#976. 
                                <a href="https://pubs.acs.org/doi/10.1021/nl201818u" target="_blank">
                                <i> " DNA-Linker-Induced Surface Assembly of Ultra Dense Parallel Single Walled Carbon Nanotube Arrays"</i></a>. 
                            </li>
                            <li class="papers">#707. 
                                <a href="https://www.ingentaconnect.com/content/asp/jnn/2007/00000007/00000006/art00006;jsessionid=5bb97m7bvwr0b.x-ic-live-03" target="_blank">
                                <i> " Understanding DNA Based Nanostructures"</i></a>. 
                            </li>
                            <li class="papers">#652. 
                                <a href="https://www.sciencedirect.com/science/article/pii/S0006349506723384?via%3Dihub" target="_blank">
                                <i> " Atomic-Level Simulations of Seeman DNA Nanostructures: The Paranemic Crossover in Salt Solution"</i></a>. 
                            </li>
                            <li class="papers">#594. 
                                <a href="https://academic.oup.com/nar/article/32/20/6047/1115579?login=true" target="_blank">
                                <i> " The stability of Seeman JX DNA topoisomers of paranemic crossover (PX) molecules as a function of crossover number"</i></a>. 
                            </li><br>
                            <div class="gather-imgs">
                                <div class="gather-img">
                                    <img alt="figure of paper no.594"
                                        src="img/research/594.jpeg"
                                        width="98%">
                                </div>
                                <div class="gather-img">
                                    <p class="figure">Figure 594. MD studies of Ned Seeman crossover structures</p>
                                </div>
                            </div><br>
                            <li class="papers">#1008. 
                                <a href="http://resolver.caltech.edu/CaltechAUTHORS:20130416-080407462" target="_blank">
                                <i> " DNA-directed assembly of multicomponent single-walled carbon nanotube devices"</i></a>. 
                            </li><br>
                        </ul><br>
                    </div>   
                </div>

                <div class="focus-menu">
                    <div class="focus-wrap">
                        <div class="focus-label">C3</div>
                        <div class="focus-title">Other Protein Projects</div>
                        <div class="focus-btn">Learn More</div>
                    </div>
                    <div class="focus-text" >
                        <span class="list-sq"></span><span class="sub-title">C3-1. Hedgehog Signalling
                        </span>
                        <ul>
                            <li class="papers">#1415. 
                                <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0246814" target="_blank">
                                <i> "Hedgehog proteins create a dynamic cholesterol interface"</i></a>. 
                                <p>During formation of the Hedgehog (Hh) signaling proteins, cooperative activities of the Hedgehog INTein (Hint) fold and Sterol Recognition Region (SRR) couple autoproteolysis to cholesterol ligation. The cholesteroylated Hh morphogens play essential roles in embryogenesis, tissue regeneration, and tumorigenesis. Despite the centrality of cholesterol in Hh function, the full structure of the Hint-SRR (“Hog”) domain that attaches cholesterol to the last residue of the active Hh morphogen remains enigmatic. </p>
                                <img alt="figure of paper no.1415"
                                    src="img/research/1415.PNG" width="90%">
                                <p class="figure">Figure 1415. Diagram of the hSHH protein. The N-terminal “Hedge” domain (residues C24-G197, grey) functions as the active Hh morphogen after cholesteroylation and N-terminal palmitoylation (palm). The C-terminal “Hog” domain, which facilitates cholesterol adduction, is comprised of the Hint fold (C198-A365, green) and the SRR (V366-S462, orange). Active site residues in the Hint fold and secondary structure motifs in the SRR are indicated.</p>
                                <p>In this work, we combined molecular dynamics simulations, photoaffinity crosslinking, and mutagenesis assays to model cholesterolysis intermediates in the human Sonic Hedgehog (hSHH) protein. Our results provide evidence for a hydrophobic Hint-SRR interface that forms a dynamic, non-covalent cholesterol-Hog complex. Using these models, we suggest a unified mechanism by which Hh proteins can recruit, sequester, and orient cholesterol, and offer a molecular basis for the effects of disease-causing hSHH mutations.
                                </p>
                            </li>
                        </ul><br>

                        <span class="list-sq"></span><span class="sub-title">C3-2. Nucleation of protein structure
                        </span>
                        <ul>
                            <li class="papers">#1532. 
                                <a href="https://www.sciencedirect.com/science/article/pii/S2666386421004537?via%3Dihub" target="_blank">
                                <i> "Complete inhibition of a polyol nucleation by a micromolar biopolymer additive"</i></a>. 
                                <p>Preventing spontaneous crystallization of supersaturated solutions by additives is of critical interest to successful process design and implementation,  We report experimentally that
                                    DAFP1, an insect AFP, completely inhibits D-mannitol nucleation. Computer simulations reveal a new mechanism for crystallization inhibition where the population of the crystal-forming conformers are selectively bound and randomized in solution by hydrogen bonding to the protein surface.</p>
                            </li>
                            <li class="papers">#1166. 
                                <a href="https://www.pnas.org/doi/full/10.1073/pnas.1601519113" target="_blank">
                                <i> "Antifreeze proteins govern the precipitation of trehalose in a freezing-avoiding insect at low temperature"</i></a>. 
                            </li>
                        </ul><br>

                        <span class="list-sq"></span><span class="sub-title">C3-3. Protein Folding
                        </span>
                        <p>Experimental studies have demonstrated that many small, single-domain proteins fold via simple two-state kinetics. We present a first principles approach for predicting these experimentally determined folding rates. Our approach is based on a nucleation-condensation folding mechanism, where the rate-limiting step is a random, diffusive search for the native tertiary topology. To estimate the rates of folding for various proteins via this mechanism, we first determine the probability of randomly sampling a conformation with the native fold topology. Next, we convert these probabilities into folding rates by estimating the rate that a protein samples different topologies during diffusive folding. This topology sampling rate is calculated using the Einstein diffusion equation in conjunction with an experimentally determined intra-protein diffusion constant. #422 applies this prediction method to the 21 topologically distinct small proteins for which two-state rate data is available. For the 18 beta-sheet and mixed alpha-beta native proteins, we predict folding rates within an average factor of 4, even though the experimental rates vary by a factor of  4 *10 **4. Interestingly, the experimental folding rates for the three four-helix bundle proteins are significantly underestimated by this approach, suggesting that proteins with significant helical content may fold by a faster, alternative mechanism.
                        </p>
                        <ul>
                            <li class="papers">#405. 
                                <a href="https://pubs.acs.org/doi/10.1021/jp983429%2B" target="_blank">
                                <i> "Protein Fold Determination from Sparse Distance Restraints:  The Restrained Generic Protein Direct Monte Carlo Method"</i></a>. 
                            </li>
                            <li class="papers">#406. 
                                <a href="https://www.pnas.org/doi/full/10.1073/pnas.96.6.2596" target="_blank">
                                <i> "The topomer-sampling model of protein folding"</i></a>. 
                            </li>
                            <li class="papers">#422. 
                                <a href="https://www.sciencedirect.com/science/article/pii/S0022283699932783?via%3Dihub" target="_blank">
                                <i> "First principles prediction of protein folding rates"</i></a>. 
                            </li><br>
                            <div class="gather-imgs">
                                <div class="gather-img">
                                    <img alt="figure of paper no.422"
                                        src="https://ars.els-cdn.com/content/image/1-s2.0-S0022283699932783-gr1_lrg.gif"
                                        width="100%">
                                </div>
                                <div class="gather-img">
                                    <p class="figure">Figure 422.  Experimental folding rate versus predicted folding rate for all 18 alpha-beta (*) and beta (~) proteins for which there is rate data. Over this
                                        data set the average error in the predicted rate is 3.7 
                                    </p>
                                </div>
                            </div>
                            <li class="papers">#680. 
                                <a href="https://onlinelibrary.wiley.com/doi/10.1002/prot.21049" target="_blank">
                                <i> "STRUCTFAST: Protein sequence remote homology detection and alignment using novel dynamic programming and profile–profile scoring"</i></a>. 
                            </li>
                        </ul><br>

                        <span class="list-sq"></span><span class="sub-title">C3-4. Glycosaminoglycan Surface Protein Interactions (Collaborations with Linda Hsieh-Wilson)
                        </span>
                        <ul>
                            <li class="papers">#1402. 
                                <a href="https://www.nature.com/articles/s41589-020-00657-7" target="_blank">
                                <i> "Sulfated glycans engage the Ang–Tie pathway to regulate vascular development"</i></a>. 
                            </li>
                            <li class="papers">#1237. 
                                <a href="https://www.pnas.org/doi/full/10.1073/pnas.1715093115" target="_blank">
                                <i> "Predicting glycosaminoglycan surface protein interactions and implications for studying axonal growth"</i></a>. 
                            </li>
                            <li class="papers">#987. 
                                <a href="https://www.science.org/doi/10.1126/science.1222278" target="_blank">
                                <i> "Phosphofructokinase 1 Glycosylation Regulates Cell Growth and Metabolism"</i></a>. 
                            </li>
                            <li class="papers">#931. 
                                <a href="https://www.pnas.org/doi/full/10.1073/pnas.1102962108" target="_blank">
                                <i> "Elucidating glycosaminoglycan–protein–protein interactions using carbohydrate microarray and computational approaches"</i></a>. 
                            </li>
                            <li class="papers">#679. 
                                <a href="https://www.nature.com/articles/nchembio810" target="_blank">
                                <i> "Sulfation patterns of glycosaminoglycans encode molecular recognition and activity"</i></a>. 
                            </li>
                        </ul>
                    </div>
                </div>

                

            </div>

                <!-- End here -->


    </div>
    </div>
</main>

<!-- Footer -->
<footer class="footer">
    <div class="footer-container container">
        <div class="upper-footer">
            <div class="footer-innerbox innerbox">
                <div class="upper-row row">
                    <div class="col" id="links"><strong>Related Links</strong></div>
                    <div class="col" id="link-item"><a href="https://www.caltech.edu/" target="_blank">California Institute of Technology</a> </div>
                    <div class="col" id="link-item"><a href="https://www.cce.caltech.edu/" target="_blank"> Caltech | Chemistry and Chemical Engineering</a> </div>
                    <div class="col" id="link-item"><a href="https://www.eas.caltech.edu/" target="_blank"> Caltech | Engineering and Applied Science</a> </div>
                    <div class="col" id="link-item"><a href="https://www.liquidsunlightalliance.org/" target="_blank">LiSA | Liquid Sunlight Alliance</a> </div>
                
                </div>
            </div>
        </div>

        <div class="lower-footer">
            <div class="footer-innerbox innerbox">
                <div class="lower-row row">      
                    <div class="col"><a href="Home.html">
                        <img id="footer-logo"
                            alt="MSC Materials and Process Simulation Center and Goddard Group of Caltech" 
                            src="img/logo-04.svg" width="90%"></a> </div>
                    <div class="col"> <strong>Location</strong><br><a href="https://www.google.com/maps/place/Beckman+Institute/@34.1390373,-118.126649,18z/data=!4m5!3m4!1s0x80c2c35861927a65:0xb6f2e635b54581a!8m2!3d34.1391999!4d-118.1265925" target="_blank">Beckman Institute (BI) <br> 1200 E California Blvd <br>Pasadena CA 91125 | Caltech</a></div>
                    <div class="col"> <strong>Contact E-mail</strong><br>
                        <a href="mailto:wagoddard3@gmail.com?">wagoddard3@gmail.com</a> <br> <a href="mailto:wag@caltech.edu?">wag@caltech.edu</a></div>
                    <div class="col" id="copyright"><strong>Copyright</strong><br>&copy 2023 MSC | Goddard Group <br> Designed & developed by <a href="mailto:dodohjin@gmail.com?">Sujin Kim</a></div>
                </div>             
            </div>
        </div>

        <button id="top" title="Go to top">Top</button>
        <script>
            let topBtn = document.querySelector("#top");
            
            window.addEventListener('scroll', function() {
                if(this.scrollY>350) {
                    topBtn.style.display = "block";
                } else{
                    topBtn.style.display = "none";
                }
            })

            topBtn.addEventListener("click", function(e) {
                e.preventDefault();
                window.scrollTo({ top:0, behavior:'smooth'});
            })
        </script>
    </div>
</footer>

<script src="js/icon.js"></script>
<script src="js/slide.js"></script>
<!-- <script src="js/map.js"></script>
<script src="js/map_tabs.js"></script>
<script src="js/pub_fetch.js"></script>
<script src="js/pub-update2.js"></script> -->
<script src="js/research-popup.js"></script>
<!-- <script src="js/members.js"></script> -->

</body>
</html>